app/models/notification.rb

```diff
@@ -55,15 +58,15 @@ class Notification < ActiveRecord::Base
   # ==========
 
   scope :active, (lambda do
-    where('starts_at <= :now and :now < expires_at', now: Time.now)
+    where('starts_at <= :now and :now < expires_at', now: Time.now).where(active: true)
```

Comment:
This threw me for a loop for a minute.
I guess an instanciated class gets an accessor `active` to refer to the model,
And then the Plan class (or an activerecord relation) will have a `active` method referring to this scope.
I think the method/attribute are defined in different contexts so there shouldnt be any conflict

@briri have you ever run into any trouble doing something like this?

Reply:
@xsrust I don't know about this. I don't think I changed this?!

Reply:
You've defined a field `active` on the model, which has the same name as this `scope :active`  
Your change to this method is just to ensure that the field is set to true, which makes sense.  I was more questioning if the overloading of that name would cause issue.  
I dont think it will, but brian's worked on more rails projects than I have:)

Reply:
Hmm. Yeah, doing `Notification.active` would invoke the scope over the attribute as far as I know. Would be interesting to test. `Notification.where(active: true)` though would refer to the attribute.

It may be cleaner to either rename the scope to something else or change this attribute to `enabled` or something.